### DISCUSSION TITLE
useLoaderData hook require an object

### DISCUSSION THREAD

**Discussion by mauscoelho** [sergiodxa/remix-utils#100] [General]
Is it expected to be like this?
Currently, I need to explicitly pass an empty object to be able to use `useLoaderData`  with default values.

![image](https://user-images.githubusercontent.com/5712300/194859693-1fe59d76-f4e0-4454-b4cd-bb82837457b2.png)
![image](https://user-images.githubusercontent.com/5712300/194859758-2f0ad6bf-83a0-4144-b724-eccbfb2fe030.png)

**Comment by sergiodxa** [comment]
This is expected, if you want to use the default values use the Remix useLoaderData hook and the json function instead since there'll be no difference.
